.github/workflows/build_deploy.yml

```diff
@@ -211,3 +211,25 @@ jobs:
           # due to a duplicate wheel being present which will ensure that the rest
           # of the wheels will be uploaded if some are uploaded manually.
           skip_existing: true
+
+  build-and-publish-init-image:
+    # We have to wait for the PyPI job since the image that we publish depends on installing
+    # the package from PyPI.
+    needs: [upload_pypi]
```

Comment:
this won't run unless the conditions that causes `uplod_pypi` runs right?

Reply:
hmm, there isn't any way to verify this until we merge right?

Reply:
there isn't, but it's effectively the same job that runs on commit to build the test image (only difference is the tag)